### DISCUSSION TITLE
[Question] - can we put persistent files in flash drives?

### DISCUSSION THREAD

**Discussion by JinSakaiM** [Monika-After-Story/MonikaModDev#8684] [QA - tech support + general] [answered]
could we?

**Comment by lunulae** [comment, marked as answer]
Transferring it or copying it to a flash drive or other external drive shouldn't cause issues. External drives can be a good place to store backups when needed.

If you mean that you'd like to run MAS from a flash drive, that is also possible, You'll need to change your install so that it uses a localized persistent--[a guide for doing so can be found here](https://github.com/Monika-After-Story/MonikaModDev/wiki/Contributing-Guidelines). 
After you've done that, you'll just need to move your existing persistent file to the new save destination in your install. Then you'll be able to move all of MAS to another drive.
